extensions/ql-vscode/src/remote-queries/sample-data.ts

```diff
@@ -0,0 +1,86 @@
+import { RemoteQuery } from './remote-query';
+import { RemoteQueryResult } from './remote-query-result';
+
+export const sampleRemoteQuery: RemoteQuery = {
+  queryName: 'Inefficient regular expression',
+  queryFilePath: '/Users/charisk/dev/vscode-codeql-starter/ql/javascript/ql/src/Performance/ReDoS.ql',
```

Comment:
Probably best not to check in your specific file path... It might be possible to point this to the right location for a general user (assuming they're also in the `vscode-codeql-starter` workspace) 🤔 

Or this could just be a dummy path 😎

Reply:
Good catch - I thought I had dummy-fied everything!

Reply:
For info, you're totally right that you generally can't have comments in JSON (there are exceptions), so I meant have the code in extension.ts commented out and leave it to to us to add the dummy command in package.json every time. Not an ideal approach, so glad we're going with checking in the command!